app/src/keymap.c

```diff
@@ -16,9 +16,11 @@ LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
 
 #include <zmk/event-manager.h>
 #include <zmk/events/position-state-changed.h>
+#include <zmk/events/layer-state-changed.h>
 #include <zmk/events/sensor-event.h>
 
-static u32_t zmk_keymap_layer_state = 0;
+typedef u32_t zmk_layers_state;
```

Comment:
This should be defined in `app/include/zmk/keymap.h` so it can be used in the definition of the `zmk_layer_state()` function.